modules/monitoring-resizing-a-persistent-volume.adoc

```diff
@@ -0,0 +1,145 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="resizing-a-persistent-volume_{context}"]
+= Resizing a persistent volume
+
+You can resize a persistent volume (PV) for monitoring components, such as Prometheus, Thanos Ruler, or Alertmanager. You need to manually expand a persistent volume claim (PVC), and then update the config map in which the component is configured.
+
+[IMPORTANT]
+====
+You can only expand the size of the PVC. Shrinking the storage size is not possible.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* *If you are configuring core {product-title} monitoring components*:
+** You have access to the cluster as a user with the `cluster-admin` cluster role.
+** You have created the `cluster-monitoring-config` `ConfigMap` object.
+** You have configured at least one PVC for core {product-title} monitoring components.
+* *If you are configuring components that monitor user-defined projects*:
+** You have access to the cluster as a user with the `cluster-admin` cluster role, or as a user with the `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project.
+** A cluster administrator has enabled monitoring for user-defined projects.
+** You have configured at least one PVC for components that monitor user-defined projects.
+
+.Procedure
+
+. Manually expand a PVC with the updated storage request. For more information, see "Expanding persistent volume claims (PVCs) with a file system" in _Expanding persistent volumes_.
```

Comment:
```suggestion
. Manually expand a PVC with the updated storage request. For more information, see below Additional resources "Expanding persistent volume claims (PVCs) with a file system" in _Expanding persistent volumes_.
```
how about it?

Reply:
Sadly, similarly to this comment thread: https://github.com/openshift/openshift-docs/pull/74645#discussion_r1596468156
I cannot even use the wording "in the Additional Resources section" here. (to be honest I do not agree with these doc guidelines, but I cannot go against them right now. Hopefully, there will be better solutions for this as we are trying to change our RH docs for the better. But right now, the way I included is the inly way to include in the text itself. :/